symfony-reproducer.md

```diff
@@ -0,0 +1,111 @@
+   already been reported and maybe a fix is already in progress. Be sure to search issues, pull requests and discussions.
+2. Ask about your issue in the `#support` channel on the [Symfony Slack](https://symfony.com/slack). This can be a good
+   way to confirm the bug or perhaps get ideas for a workaround. Keep in mind that users who respond in this channel are
+   volunteers. Don't be discouraged if you don't get an answer.
+3. Do you think you know how to fix the issue? Open a PR! See the
+   [Symfony documentation](https://symfony.com/doc/current/contributing/code/pull_requests.html) to help with this.
+   Be sure to add a test if possible as this will likely be asked of you during the review.
+4. Not sure how to fix? Are you able to create a failing test? Same as the above step, create a PR that demonstrates
+   the bug in a failing test. Even if it doesn't include a fix, it's highly likely it will get reviewed and confirmed.
+   You'll also likely get tips on how to fix!
+5. Open a [bug report issue](https://github.com/symfony/symfony/issues/new/choose) and clearly explain the issue
+   you are facing. Initially including a reproducer (as we'll build below) is a great way to make your issue stand out!
+
+> [!TIP]
+> In general, pull requests (like described in steps 3 and 4 above) are more likely to catch the eye
+> of maintainers than issues (even a PR that only includes a failing test).
+
+After you've created an issue (step 5 above), a maintainer (or even another contributor) will hopefully start
+a conversation to discuss the bug further. It's possible the bug is hard to describe or hard for maintainers to
+understand. A common thing they may ask is to _create a reproducer_. What's this? How do you do this? In my
+experience as a maintainer, when asking for a reproducer, this usually ends the discussion as it perhaps
+seems too hard or time-consuming for the contributor. This is not the case! Let's look at how to do this!
+
+## What is a Reproducer
+
+A reproducer is a standard, fresh, Symfony application that, aside from the initial Symfony app skeleton code,
+includes only the code to reproduce the bug you are seeing. Maintainers love these as it makes it super easy
+to see the same bug as you. They can clone it and test it locally to confirm and test possible fixes.
+
+> [!TIP]
+> Just the process of creating a reproducer is a great learning experience and helps narrow down the actual
+> bug. Many times, working on a reproducer will resolve your issue as you may find it's not a bug at all but
+> just a misconfiguration or misuse of a feature.
+
+## Create a Reproducer
+
+The easiest way to create a reproducer is using the [Symfony CLI](https://symfony.com/download):
+
+```bash
+symfony new --webapp my-reproducer
+```
+
+This new app will be using the latest stable version of Symfony. If you need to target
+a specific Symfony version, use the `--version` option:
```

Comment:
Maybe the next two examples aren't required?

I wanted to show two things:
1. Bugs are fixed on the lowest maintained Symfony branch
2. We love when users try out latest dev versions to help find bugs before a release

I think 2 is for sure out of scope of this post and perhaps 1 as well. I think we want to get users up and running with a reproducer as quick as possible. Feels like explaining these Symfony nuances muddy's the waters a big...

But I'm curious on your thoughts.

Reply:
I think the example one is useful, but the second could be removed at least the `bash` block. However, I don't think that item 1 is reflected. From my point of view, you're just showing how to select a Symfony version but it's not clear that I (as a contributor) should use the lowest version possible

Reply:
Hmm, my thinking here is: as a new user, using a lower version of Symfony than your app could complicate things when creating your reproducer might require features that aren't available in that Symfony version.